src/components/Jobs/Jobs.js

```diff
@@ -231,7 +227,7 @@ const Jobs = () => {
                       : '',
                     variant: TERTIARY_BUTTON,
                     disabled: !appStore.frontendSpec.jobs_dashboard_url,
-                    onClick: () => handleMonitoring()
+                    onClick: handleMonitoring
```

Comment:
we have a lot of places where we're using onClick: () => handleMonitoring(), should it be changed only here?